tasks/configure-Debian.yml

```diff
@@ -38,6 +38,23 @@
   notify: restart apache
   when: apache_create_vhosts
 
+- name: Configure Apache (Add listen port for every vhost).
+  blockinfile:
+    dest: "{{ apache_server_root }}/ports.conf"
```

Comment:
Instead of duplicating this task for each configure-[OS] file, can you put it in one place, and maybe use a new variable like `{{ apache_ports_conf_dir }}` to allow each OS to have a default?